binder/src/main/java/io/grpc/binder/BinderChannelBuilder.java

```diff
@@ -224,6 +227,23 @@ public BinderChannelBuilder inboundParcelablePolicy(
     return this;
   }
 
+  /** 
+    * Enables strict lifecycle management. This should be called by processes with elevated procrank, where
+    * reliance on idle timers is inappropriate for lifecycle management.
```

Comment:
Seems this should include more details, like:
> Disables the channel idle timeout and prevents it from being enabled. This allows a centralized application method to configure the channel builder and return it, without worrying about another part of the application accidentally enabling the idle timeout.

Reply:
Sorry for the long delay, done.